test/files/Onda.jl/LICENSE.md

```diff
@@ -1,22 +1,21 @@
 The Onda.jl package is licensed under the MIT "Expat" License:
 
 > Copyright (c) 2019: Beacon Biosignals, Inc.
->
+> 
```

Comment:
is this whitespace? kinda weird that's added right?

Reply:
Yes, it is. I could open up an issue in CommonMark?